### PR TITLE
feat(store-sync): adjust DB schema/table names for consistency

### PR DESCRIPTION
**Pull request by holic**
part of #2173 

- moves postgres from `{address}__{namespace}.{tableName}` to `{address}.{namespace}__{tableName}`
- moves sqlite to use snake case for table/column names to match postgres

### CODE REVIEW

**Review by yonadaaa — COMMENTED** [review]
The reasoning makes sense! I worry this might be less useable though, because the most useful "divisions" (ie. pg namespaces) we can have are MUD namespaces. 

For example, with the Sky Strife analytics indexer, the indexer already has a specific world address, and it's useful to `SEARCH_PATH` a specific namespace to save rewriting long table names: https://github.com/latticexyz/skystrife-public/blob/main/sql-queries/WalletMatchesJoined.sql#L1-L2

basically does this best take advantage of Postgres namespaces?

**Comment by holic** [comment]
since sky strife uses root namespace, you won't need to do anything differently here except remove the `__` suffix from the schema
